### PR TITLE
Fix eth-bot automerging & configure kodiak

### PR DESCRIPTION
**Pull request by Pandapip1**
This fixes the "pull request unstable" bug.

~~**The `EIP Auto-Merge Bot` check MUST be added to the branch protection required checks before this can be safely merged.**~~
Never mind, it already is. Yay!

### CODE REVIEW

File: .kodiak.toml [hunk 1]
```diff
@@ -4,12 +4,23 @@ version = 1
...
+# Add useful information & co-authors to the PR commit body
+[merge.message]
+include_coauthors = true
+include_pull_request_author = true
+include_pull_request_url = true
+
+# Keep dependencies up-to-date automatically 
+[merge.automerge_dependencies]
+versions = ["minor", "patch"] # do not automerge major version upgrades
+usernames = ["dependabot"]
```
Comment (SamWilsn) [inline .kodiak.toml:22]: Does this bypass the need for an editor to approve? I dislike that from a supply chain attack perspective.

File: .kodiak.toml [hunk 2]
```diff
@@ -4,12 +4,23 @@ version = 1
...
+
+# Automatically keep PRs ahead of the main branch
+[update]
+always = true
```
Comment (SamWilsn) [inline .kodiak.toml:26]: Bots shouldn't edit commits on other people's PRs.

**Review by SamWilsn — CHANGES_REQUESTED** [review]

File: .kodiak.toml (hunk 1 above)
Comment (Pandapip1) [reply .kodiak.toml:22]: Yes it does. I'll remove it then.

File: .kodiak.toml (hunk 2 above)
Comment (SamWilsn) [reply .kodiak.toml:21]: Hm, what does `update` mean in this context? I assumed it meant rebase on top of the default branch.

Comment (Pandapip1) [reply .kodiak.toml:21]: That's exactly it, but it fundamentally doesn't edit the authors' commits.

Comment (SamWilsn) [reply .kodiak.toml:21]: That is literally fundamentally editing the authors' commits. The hash will change, meaning they'll need to force pull. That's extremely impolite.

Comment (Pandapip1) [reply .kodiak.toml:21]: Fair. I'll remove it.

**Comment by Pandapip1** [comment]
YOLO I guess

**Pull request merged**